Update manual pages to add details to --probes to contrast with --retries

phillip-stephens opened the pull request:
- added details to `--probes` to contrast with `--retries`
- re-generated manual pages with `make manpages`
- added `.DS_Store` to `.gitignore`. Looks like it can be auto-generated on MacOS, don't want that slipping into the repo

zakird commented:
This broadly looks good to me. We should update here to say that probes isn't number of packets per _IP_ but per target _IP/port_ pair. (This is my fault, but we should update it).

Can you make sure that we're also updating the `--help` text as well? Unfortunately the two are maintained separately. I think the text there is generally much shorter, but I did notice that under `probes` definition there it says packets per IP, which should be packets per ip/port too.

phillip-stephens commented:
@zakird I addressed your concerns and updated the wiki with the `multi-port` usages. If you could re-review whenever you have a chance, that'd be great!
